### ISSUE TITLE
Crash when creating world.

### ISSUE THREAD

**Issue by 1-rick** [PG85/OpenTerrainGenerator#284]
Full crash dump here: https://pastebin.com/54DzYeTj

First few lines:

Description: Exception in server tick loop

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Open Terrain Generator (openterraingenerator)
Caused by: java.lang.NullPointerException
	at com.pg85.otg.forge.dimensions.OTGDimensionManager.UnloadAllCustomDimensionData(OTGDimensionManager.java:409)
	at com.pg85.otg.forge.ForgeWorld.<init>(ForgeWorld.java:149)

Lately I've been getting this almost every time I generate a new world.  This is MC 1.12.2, Forge 14.23.5.2838, and OTG v6.  I updated a few other mods and added OTG Void V4 and OTG Skylands V5 on the same day I started getting these crashes.

**Comment by PG85** [comment]
Hey @1-rick, thanks for reporting this. Could you try OTG v7 (available on the OTG Discord #OpenTerrainGen's otg-dev channel) and see if that fixes the issue?

**Comment by 1-rick** [comment]
I tried v7, and got a *different* crash.  I will disable Mo' Villages and try again and post an update.

Description: Exception in server tick loop
java.lang.ClassCastException: com.pg85.otg.forge.generator.structure.OTGVillageGen cannot be cast to net.minecraft.world.gen.structure.MapGenVillage
	at com.piggywiggy3.movillages.handlers.MapGenEventHandler.onMapGenInit(MapGenEventHandler.java:24)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_482_MapGenEventHandler_onMapGenInit_InitMapGenEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
	at net.minecraftforge.event.terraingen.TerrainGen.getModdedMapGen(TerrainGen.java:48)
	at com.pg85.otg.forge.ForgeWorld.provideWorldInstance(ForgeWorld.java:988)
	at com.pg85.otg.forge.WorldLoader.getOrCreateForgeWorld(WorldLoader.java:270)
	at com.pg85.otg.forge.OTGWorldType.getBiomeProvider(OTGWorldType.java:140)
	at net.minecraft.world.WorldProvider.func_76572_b(WorldProvider.java:58)
	at net.minecraft.world.WorldProvider.func_76558_a(WorldProvider.java:40)
	at net.minecraft.world.WorldServer.<init>(WorldServer.java:116)
	at net.optifine.override.WorldServerOF.<init>(WorldServerOF.java:27)
	at net.minecraft.server.integrated.IntegratedServer.func_71247_a(IntegratedServer.java:94)
	at net.minecraft.server.integrated.IntegratedServer.func_71197_b(IntegratedServer.java:209)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486)
	at java.lang.Thread.run(Thread.java:745)

**Comment by 1-rick** [comment]
Wow.  Completely different error this time, with Mo' Villages disabled.  BTW, when I start the game up again after the crash I can enter the world with no problems.

---- Minecraft Crash Report ----

WARNING: coremods are present:
  IELoadingPlugin (ImmersiveEngineering-core-0.12-89.jar)
  PhosphorFMLLoadingPlugin (phosphor-1.12.2-0.2.4+build46-universal.jar)
  ForgelinPlugin (Forgelin-1.7.4.jar)
  HCASM (HammerCore-1.12.2-2.0.0.4.jar)
  OTGCorePlugin (OTG-Core.jar)
  CoreMod (Aroma1997Core-1.12.2-2.0.0.0.jar)
Contact their authors BEFORE contacting forge

// Hi. I'm Minecraft, and I'm a crashaholic.

Time: 7/30/19 4:02 PM
Description: Ticking entity

java.lang.NoClassDefFoundError: com/pg85/otg/forge/dimensions/WorldProviderOTG
	at com.pg85.otg.forge.asm.OTGHooks.getGravityFactor(OTGHooks.java:26)
	at net.minecraft.entity.EntityLivingBase.func_191986_a(EntityLivingBase.java:1977)
	at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:2393)
	at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:678)
	at net.minecraft.entity.EntityAgeable.func_70636_d(EntityAgeable.java:178)
	at net.minecraft.entity.passive.EntityAnimal.func_70636_d(SourceFile:43)
	at net.minecraft.entity.passive.EntityChicken.func_70636_d(SourceFile:83)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2179)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:377)
	at net.minecraft.world.World.func_72866_a(World.java:1993)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)
	at net.minecraft.world.World.func_72870_g(World.java:1955)
	at net.minecraft.world.World.func_72939_s(World.java:1759)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:252)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ClassNotFoundException: com.pg85.otg.forge.dimensions.WorldProviderOTG
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 19 more
Caused by: java.lang.NullPointerException


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at com.pg85.otg.forge.asm.OTGHooks.getGravityFactor(OTGHooks.java:26)
	at net.minecraft.entity.EntityLivingBase.func_191986_a(EntityLivingBase.java:1977)
	at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:2393)
	at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:678)
	at net.minecraft.entity.EntityAgeable.func_70636_d(EntityAgeable.java:178)
	at net.minecraft.entity.passive.EntityAnimal.func_70636_d(SourceFile:43)
	at net.minecraft.entity.passive.EntityChicken.func_70636_d(SourceFile:83)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2179)
	at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:377)
	at net.minecraft.world.World.func_72866_a(World.java:1993)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)
	at net.minecraft.world.World.func_72870_g(World.java:1955)

-- Entity being ticked --
Details:
	Entity Type: minecraft:chicken (net.minecraft.entity.passive.EntityChicken)
	Entity ID: 3
	Entity Name: Chicken
	Entity's Exact location: 4.50, 64.00, 41.50
	Entity's Block location: World: (4,64,41), Chunk: (at 4,4,9 in 0,2; contains blocks 0,0,32 to 15,255,47), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Entity's Momentum: 0.00, 0.00, 0.00
	Entity's Passengers: []
	Entity's Vehicle: ~~ERROR~~ NullPointerException: null
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1759)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)

-- Affected level --
Details:
	Level name: Moosh
	All players: 0 total; []
	Chunk stats: ServerChunkCache: 625 Drop: 0
	Level seed: -257105894062709242
	Level generator: ID 07 - OTG, ver 0. Features enabled: true
	Level generator options: OpenTerrainGenerator
	Level spawn location: World: (29,63,-45), Chunk: (at 13,3,3 in 1,-3; contains blocks 16,0,-48 to 31,255,-33), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,0,-512 to 511,255,-1)
	Level time: 1 game time, 1 day time
	Level dimension: 0
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 64053 (now: false), thunder time: 122085 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:252)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Thread.java:745)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1304933296 bytes (1244 MB) / 3355443200 bytes (3200 MB) up to 4294967296 bytes (4096 MB)
	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.42 Powered by Forge 14.23.5.2838 Optifine OptiFine_1.12.2_HD_U_E3 55 mods loaded, 55 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State  | ID                      | Version           | Source                                            | Signature                                |
	|:------ |:----------------------- |:----------------- |:------------------------------------------------- |:---------------------------------------- |
	| LCHIJA | minecraft               | 1.12.2            | minecraft.jar                                     | None                                     |
	| LCHIJA | mcp                     | 9.42              | minecraft.jar                                     | None                                     |
	| LCHIJA | FML                     | 8.0.99.99         | forge-1.12.2-14.23.5.2838.jar                     | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| LCHIJA | forge                   | 14.23.5.2838      | forge-1.12.2-14.23.5.2838.jar                     | e3c3d50c7c986df74c645c0ac54639741c90a557 |
	| LCHIJA | otgcore                 | 1.12.2 - v7       | minecraft.jar                                     | None                                     |
	| LCHIJA | actuallyadditions       | 1.12.2-r148       | ActuallyAdditions-1.12.2-r148.jar                 | None                                     |
	| LCHIJA | jei                     | 4.15.0.268        | jei_1.12.2-4.15.0.268.jar                         | None                                     |
	| LCHIJA | appleskin               | 1.0.9             | AppleSkin-mc1.12-1.0.9.jar                        | None                                     |
	| LCHIJA | aroma1997core           | 2.0.0.0           | Aroma1997Core-1.12.2-2.0.0.0.jar                  | dfbfe4c473253d8c5652417689848f650b2cbe32 |
	| LCHIJA | aroma1997sdimension     | 1.3.0.0.b37       | Aroma1997s-Dimensional-World-1.12-1.3.0.0.b37.jar | None                                     |
	| LCHIJA | aromabackup             | 2.1.1.4           | AromaBackup-1.12.2-2.1.1.4.jar                    | dfbfe4c473253d8c5652417689848f650b2cbe32 |
	| LCHIJA | aromabackuprecovery     | 2.1.1.4           | AromaBackup-1.12.2-2.1.1.4.jar                    | dfbfe4c473253d8c5652417689848f650b2cbe32 |
	| LCHIJA | betterchests            | 3.0.0.0.b83       | BetterChests-1.12.2-3.0.0.0.b83.jar               | dfbfe4c473253d8c5652417689848f650b2cbe32 |
	| LCHIJA | openterraingenerator    | v7                | OpenTerrainGenerator-1.12.2_-_v7_DEV_r26.jar      | None                                     |
	| LCHIJA | biomebundle             | 5.1               | Biome_Bundle-1.12.2-v6.1.jar                      | None                                     |
	| LCHIJA | claybucket              | 1.1               | ClayBucket-1.12-1.1.jar                           | None                                     |
	| LCHIJA | codechickenlib          | 3.2.2.353         | CodeChickenLib-1.12.2-3.2.2.353-universal.jar     | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| LCHIJA | redstoneflux            | 2.0.2             | RedstoneFlux-1.12-2.0.2.3-universal.jar           | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| LCHIJA | cofhcore                | 4.5.3             | CoFHCore-1.12.2-4.5.3.20-universal.jar            | None                                     |
	| LCHIJA | cofhworld               | 1.2.0             | CoFHWorld-1.12.2-1.2.0.5-universal.jar            | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| LCHIJA | mantle                  | 1.12-1.3.2.24     | Mantle-1.12-1.3.2.24.jar                          | None                                     |
	| LCHIJA | tconstruct              | 1.12.2-2.10.1.87  | TConstruct-1.12.2-2.10.1.87.jar                   | None                                     |
	| LCHIJA | extrautils2             | 1.0               | extrautils2-1.12-1.8.5.jar                        | None                                     |
	| LCHIJA | fastleafdecay           | v14               | FastLeafDecay-v14.jar                             | None                                     |
	| LCHIJA | forgelin                | 1.7.4             | Forgelin-1.7.4.jar                                | None                                     |
	| LCHIJA | silentlib               | 2.2.18            | SilentLib-1.12-2.2.18-100.jar                     | None                                     |
	| LCHIJA | funores                 | 1.5.7             | FunOres-1.12-1.5.7-118.jar                        | None                                     |
	| LCHIJA | hammercore              | 2.0.0.4           | HammerCore-1.12.2-2.0.0.4.jar                     | 4d7b29cd19124e986da685107d16ce4b49bc0a97 |
	| LCHIJA | waila                   | 1.8.26            | Hwyla-1.8.26-B41_1.12.2.jar                       | None                                     |
	| LCHIJA | teslacorelib            | 1.0.14            | tesla-core-lib-1.12.2-1.0.14.jar                  | d476d1b22b218a10d845928d1665d45fce301b27 |
	| LCHIJA | industrialforegoing     | 1.12.2-1.12.2     | industrialforegoing-1.12.2-1.10.3-181.jar         | None                                     |
	| LCHIJA | ironchest               | 1.12.2-7.0.40.824 | ironchest-1.12.2-7.0.40.824.jar                   | None                                     |
	| LCHIJA | harvestcraft            | 1.12.2u           | Pam's+HarvestCraft+1.12.2u.jar                    | None                                     |
	| LCHIJA | jehc                    | 1.12.2            | JEHC-1.12.2-1.6.1.0.jar                           | None                                     |
	| LCHIJA | journeymap              | 1.12.2-5.5.2      | journeymap-1.12.2-5.5.2.jar                       | None                                     |
	| LCHIJA | moarboats               | 4.1.1.0           | moarboats-1.12.2-4.1.1.0.jar                      | None                                     |
	| LCHIJA | natura                  | 1.12.2-4.3.2.49   | natura-1.12.2-4.3.2.49.jar                        | None                                     |
	| LCHIJA | nei                     | 2.4.2             | NotEnoughItems-1.12.2-2.4.2.240-universal.jar     | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
	| LCHIJA | otgskylands             | 1.0               | OTG-Skylands-1.12.2-v5.jar                        | None                                     |
	| LCHIJA | otgvoid                 | 1.0               | OTG-Void-1.12.2-v4.jar                            | None                                     |
	| LCHIJA | solarfluxreborn         | 3.60a             | SolarFluxReborn_1.12.2-3.60a.jar                  | None                                     |
	| LCHIJA | thermalfoundation       | 2.5.0             | ThermalFoundation-1.12.2-2.5.0.19-universal.jar   | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| LCHIJA | thermalexpansion        | 5.5.0             | ThermalExpansion-1.12.2-5.5.0.29-universal.jar    | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| LCHIJA | thermaldynamics         | 2.5.1             | ThermalDynamics-1.12.2-2.5.1.14-universal.jar     | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| LCHIJA | tp                      | 3.2.29            | tinyprogressions-1.12.2-3.3.29-Release.jar        | None                                     |
	| LCHIJA | twilightforest          | 3.7.424           | twilightforest-1.12.2-3.7.424-universal.jar       | None                                     |
	| LCHIJA | vanillasatchels         | 1.0.0             | VanillaSatchels-1.12.2-1.0.0.1-universal.jar      | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| LCHIJA | vanillatools            | 1.0.0             | VanillaTools-1.12.2-1.0.0.1-universal.jar         | 8a6abf2cb9e141b866580d369ba6548732eff25f |
	| LCHIJA | veinminer               | 0.38.2            | VeinMiner-1.12-0.38.2.647+b31535a.jar             | None                                     |
	| LCHIJA | veinminermodsupport     | 0.38.2            | VeinMiner-1.12-0.38.2.647+b31535a.jar             | None                                     |
	| LCHIJA | wawla                   | 2.5.257           | Wawla-1.12.2-2.5.257.jar                          | d476d1b22b218a10d845928d1665d45fce301b27 |
	| LCHIJA | weeeflowers             | 1.12.2b           | Weee!+Flowers+1.12.2b.jar                         | None                                     |
	| LCHIJA | immersiveengineering    | 0.12-89           | ImmersiveEngineering-0.12-89.jar                  | 4cb49fcde3b43048c9889e0a3d083225da926334 |
	| LCHIJA | phosphor-lighting       | 1.12.2-0.2.4      | phosphor-1.12.2-0.2.4+build46-universal.jar       | f0387d288626cc2d937daa504e74af570c52a2f1 |
	| LCHIJA | teslacorelib_registries | 1.0.14            | tesla-core-lib-1.12.2-1.0.14.jar                  | None                                     |

	Loaded coremods (and transformers): 
IELoadingPlugin (ImmersiveEngineering-core-0.12-89.jar)
  blusunrize.immersiveengineering.common.asm.IEClassTransformer
PhosphorFMLLoadingPlugin (phosphor-1.12.2-0.2.4+build46-universal.jar)
  
ForgelinPlugin (Forgelin-1.7.4.jar)
  
HCASM (HammerCore-1.12.2-2.0.0.4.jar)
  com.zeitheron.hammercore.asm.HammerCoreTransformer
OTGCorePlugin (OTG-Core.jar)
  com.pg85.otg.forge.asm.OTGClassTransformer
CoreMod (Aroma1997Core-1.12.2-2.0.0.0.jar)
  
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	Pulsar/tconstruct loaded Pulses: 
		- TinkerCommons (Enabled/Forced)
		- TinkerWorld (Enabled/Not Forced)
		- TinkerTools (Enabled/Not Forced)
		- TinkerHarvestTools (Enabled/Forced)
		- TinkerMeleeWeapons (Enabled/Forced)
		- TinkerRangedWeapons (Enabled/Forced)
		- TinkerModifiers (Enabled/Forced)
		- TinkerSmeltery (Enabled/Not Forced)
		- TinkerGadgets (Enabled/Not Forced)
		- TinkerOredict (Enabled/Forced)
		- TinkerIntegration (Enabled/Forced)
		- TinkerFluids (Enabled/Forced)
		- TinkerMaterials (Enabled/Forced)
		- TinkerModelRegister (Enabled/Forced)
		- wailaIntegration (Enabled/Not Forced)

	Hammer Core Information: FluidDictionary Info: Registered 2 fluids.
OreDictionary Info: Registered 1950 names; 2509 stacks.
Configuration Info: IyBDb25maWd1cmF0aW9uIGZpbGUNCg0KY2xpZW50IHsNCiAgICAjIFNob3VsZCBIYW1tZXIgQ29yZSByZXBsYWNlIGVuY2hhbnRtZW50IGdsaW50IGNvbG9yIHdpdGggbW9yZSBtYXRjaGluZyBjb2xvcnMgZm9yIHNvbWUgdmFuaWxsYSBpdGVtcz8gW2RlZmF1bHQ6IHRydWVdDQogICAgQjoiQ3VzdG9tIEVuY2hhbnRtZW50IENvbG9ycyBmb3IgVmFuaWxsYSBJdGVtcyI9dHJ1ZQ0KDQogICAgIyBTaG91bGQgSGFtbWVyIENvcmUgcG9ydGZvcndhcmQgeW91ciBsb2NhbCB3b3JsZCB3aGVuIHlvdSBzaGFyZSB3aXRoIGl0IHRvIExBTj8NCiAgICAjIFRoaXMgZmVhdHVyZSBkb2Vzbid0IHdvcmsgb24gYWxsIGNsaWVudHMsIHlvdSBhcmUgZ29pbmcgdG8gZ2V0IGEgbWVzc2FnZSBpbiBjaGF0IGlmIGl0IHdhcyBzdWNjZXNzZnVsL3Vuc3VjY2Vzc2Z1bCBbZGVmYXVsdDogdHJ1ZV0NCiAgICBCOiJMQU4gVVBuUCI9dHJ1ZQ0KDQogICAgIyBSZXBsYWNlIHZhbmlsbGEgZnVybmFjZSBhbmQgYnJld2luZyBzdGFuZCBwcm9ncmVzcyBiYXJzIHRvIHVzZSBzbW9vdGggcmVuZGVyaW5nPyBbZGVmYXVsdDogdHJ1ZV0NCiAgICBCOiJTbW9vdGggVmFuaWxsYSBHdWlzIj10cnVlDQp9DQoNCg0KaXdyZW5jaCB7DQogICAgIyBXaGF0IGlzIHRoZSBjb2xvciB3aGVuIHRoZSBob3ZlcmVkIGJsb2NrIGlzIHdyZW5jaGFibGU/IChjb2xvciBlbmNvZGVkIGluIGhleCBmb3JtIHN1Y2ggYXMgUlJHR0JCKQ0KICAgIFM6R3JlZW49MjJGRjIyDQoNCiAgICAjIFdoYXQgaXMgdGhlIGNvbG9yIHdoZW4gdGhlIGhvdmVyZWQgYmxvY2sgaXMgTk9UIHdyZW5jaGFibGU/IChjb2xvciBlbmNvZGVkIGluIGhleCBmb3JtIHN1Y2ggYXMgUlJHR0JCKQ0KICAgIFM6UmVkPUZGMjIyMg0KfQ0KDQoNCiJ2YW5pbGxhIGltcHJvdmVtZW50cyIgew0KICAgICMgU2hvdWxkIEhhbW1lciBDb3JlIGZvcmNlLXNwYXduIEVuZGVyIERyYWdvbiBFZ2cgb24gRW5kZXIgRHJhZ29uIGRlYXRoPyBbZGVmYXVsdDogdHJ1ZV0NCiAgICBCOiJBbHdheXMgU3Bhd24gRHJhZ29uIEVnZyI9dHJ1ZQ0KfQ0KDQoNCg==
Pastebin Connection: Okay.
	Pulsar/natura loaded Pulses: 
		- NaturaCommons (Enabled/Forced)
		- NaturaOverworld (Enabled/Not Forced)
		- NaturaNether (Enabled/Not Forced)
		- NaturaDecorative (Enabled/Not Forced)
		- NaturaTools (Enabled/Not Forced)
		- NaturaEntities (Enabled/Not Forced)
		- NaturaOredict (Enabled/Forced)
		- NaturaWorld (Enabled/Not Forced)

	Profiler Position: N/A (disabled)
	Player Count: 0 / 8; []
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'

**Comment by 1-rick** [comment]
I got suspicious, so I backed up my Minecraft installation, and created a new one with nothing but the currently-available-on-Curseforge versions of OTG, Biome Bundle, OTG Void, and OTG Skylands, went to do a new world, cloned the default "Biome Bundle" one, and then created and destroyed it several times and was unable to replicate the crash.  I think this probably means it's an interaction with one of my other 50 or so mods.

Go ahead and close this issue.  If I happen to figure out a way to minimally reproduce it I will open a new one.
